src/dev/code_coverage/docs/ingest_code_coverage_readme.md

```diff
@@ -15,17 +15,37 @@ https://kibana-ci.elastic.co/job/elastic+kibana+qa-research/
  
```

Comment:
CI links in this file are no longer valid. Could you update it to have a https://buildkite.com/elastic/kibana-code-coverage-main ?